target/linux/ramips/dts/mt7620a_netis_wf2770.dts

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "netis,wf2770", "ralink,mt7620a-soc";
+	model = "NETIS WF2770";
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan {
+			label = "wf2770:blue:wlan";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		rfkill {
+			label = "rfkill";
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+
+		led {
+			label = "led";
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_LIGHTS_TOGGLE>;
```

Comment:
Is that doing anything with OpenWrt?

Reply:
By default it does nothing (for now). But a user can utilize it by creating `/etc/rc.button/lights_toggle` and add some custom script. Maybe "ledswitch" or "ledbutton" is more intuitive?

Reply:
led-toogle?

Reply:
That sounds better to me. I'll take it, thanks.

Reply:
Actually, I wasn't aware that KEY_LIGHTS_TOGGLE is available in gpio-button-hotplug.c, because I never encountered it before. But since it's there, is seems fine to me.